src/fetch_git.h

```diff
@@ -19,7 +19,6 @@
 #define _RESTRAINT_FETCH_GIT_H
 
 #define GIT_PORT 9418
-#define GIT_BRANCH "master"
```

Comment:
You can still have a define here..

#define GIT_BRANCHES "main", "master"

Reply:
Good suggestion! `GIT_BRANCHES` is added to file `src/fetch_git.h`.